### PR TITLE
feat(icons): Revise archive icon

### PR DESCRIPTION
**Pull request by jonsch318**
I personally find the archive icon in the popup to be not quite fitting / out of place to the rest of the icons. The stroke is too small on the top. The idea behind the icon (putting mail into a drawer is great) but it cannot fit into the 16x-16x or 32x-32x pixel space. **I** created a new one that is similar to the one used by lucide icons.
Old:
image

New:

image

I tried to match the style of the previous icons including border thickness and radius and the a single path as a svg.

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
Yeah, I also agree the box icon fits better for archive, although the current icon comes from GNOME and it's similar to the _Inbox_ and _Sent_ icons, and IMO looks fine in the sidebar context.

That said, I'm fine about changing the icon, but I would prefer to use an icon from GNOME's [icon-development-kit](https://gitlab.gnome.org/Teams/Design/icon-development-kit), since they have a icon for the same object you're suggesting, called `shoe-box-symbolic`:
 
Captura desde 2024-11-21 12-03-19

You can download the icon using the [Icon Library](https://flathub.org/apps/org.gnome.design.IconLibrary) app to update the PR.

**Comment by jonsch318** [comment]
This is the shoe-box icon. Your choice
image

File: theme/icons/mail-archive-symbolic.svg [hunk 1]
```diff
@@ -1,5 +1,2 @@
...
-</g></svg>
+<svg xmlns="http://www.w3.org/2000/svg" height="16px" viewBox="0 0 16 16" width="16px"><g fill="#222222"><path d="m 14 3 v 2 h -12 v -2 z m -14 0 v 2 c 0 1.109375 0.890625 2 2 2 h 12 c 1.109375 0 2 -0.890625 2 -2 v -2 c 0 -1.109375 -0.890625 -2 -2 -2 h -12 c -1.109375 0 -2 0.890625 -2 2 z m 0 0"/><path d="m 13 7 v 7 h -10 v -7 z m -12 0 v 7 c 0 1.109375 0.890625 2 2 2 h 10 c 1.109375 0 2 -0.890625 2 -2 v -7 c 0 -1.109375 -0.890625 -2 -2 -2 h -10 c -1.109375 0 -2 0.890625 -2 2 z m 0 0"/><path d="m 6 8 h 4 v 1 h -4 z m 0 0"/></g></svg>
```
Comment (rafaelmardojai) [inline theme/icons/mail-archive-symbolic.svg:2]: Please change the `fill="#222222"` and add `fill="context-fill" stroke="context-stroke" fill-opacity="context-fill-opacity" stroke-opacity="context-stroke-opacity"`. This is required for changing the icon color in CSS.

Comment (jonsch318) [reply theme/icons/mail-archive-symbolic.svg]: Forgot on the new icons. Have done that now

**Comment by rafaelmardojai** [comment]
Looks good! Thanks for taking the time!

**Pull request merged**